### PR TITLE
Feature/alcs 903

### PR DESCRIPTION
**Pull request by lstod**
Added inactive local governments and mapping to FN gov

### CODE REVIEW

**Comment by lstod** [comment]
merge to develop now

File: bin/migrate-oats-data/sql/application-etl-table-create.sql [hunk 1]
```diff
@@ -36,19 +36,44 @@ INSERT INTO
...
     SELECT 
-            atl.application_id
+            atl.application_id as app_id
     FROM application_type_lookup atl
         GROUP BY atl.application_id 
-        HAVING count(application_id) > 1;
+        HAVING count(application_id) > 1
+        ),
+  duplicate_ids as (      
+    select alr_application_id, alr_appl_component_id, alr_change_code
```
Comment (dhaselhan) [inline bin/migrate-oats-data/sql/application-etl-table-create.sql:57]: Keep your SQL commands as uppercase, makes it easier to read.

File: bin/migrate-oats-data/sql/insert-batch-application.sql [hunk 2]
```diff
@@ -48,6 +48,7 @@ alcs_gov as(
         join alcs.application_local_government alg on
    (case
    	when oats_gov.oats_gov_name LIKE 'Islands Trust%' then 'Islands Trust'
+    when oats_gov.oats_gov_name LIKE 'Sliammon%' then 'Tla''amin Nation'
```
Comment (dhaselhan) [inline bin/migrate-oats-data/sql/insert-batch-application.sql:51]: spacing

File: bin/migrate-oats-data/sql/insert-batch-application.sql [hunk 3]
```diff
@@ -76,7 +77,10 @@ application_type_lookup AS (
         oaac.alr_change_code AS code
     FROM
         oats.oats_alr_appl_components AS oaac
-        JOIN oats.oats_alr_change_codes oacc ON oaac.alr_change_code = oacc.alr_change_code   
+        JOIN oats.oats_alr_change_codes oacc ON oaac.alr_change_code = oacc.alr_change_code
+        LEFT JOIN oats.alcs_etl_application_exclude aee ON oaac.alr_appl_component_id = aee.component_id
+    where aee.component_id is null
```
Comment (dhaselhan) [inline bin/migrate-oats-data/sql/insert-batch-application.sql:82]: spacing, where should be capitalized

File: services/apps/alcs/src/providers/typeorm/migrations/1687546406729-seed_add_adl_loc_gov.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	 ('0136a261-6237-4332-ae6a-1c7d7f49f0e8',NULL,NOW(),NULL,'migration_seed',NULL,'Village of Chase','INTR',NULL,false,true,'{}'),
+	 ('5b5b05d3-df2e-403c-9cf5-3c52c68ab559',NULL,NOW(),NULL,'migration_seed',NULL,'Village of Cumberland','ISLR',NULL,false,true,'{}'),
+	 ('e0684ddf-fa61-47c2-87b3-1c72bed5c365',NULL,NOW(),NULL,'migration_seed',NULL,'District of Fort St. James','NORR',NULL,false,true,'{}'),
+	 ('5dc1a21a-a740-402e-a56f-8f6c85991739',NULL,NOW(),NULL,'migration_seed',NULL,'District of Wells','INTR',NULL,false,true,'{}'),
+	 ('630ee8b6-6bb0-4357-a1e8-d813c3b89433',NULL,NOW(),NULL,'migration_seed',NULL,'Village of Gold River','ISLR',NULL,false,true,'{}'),
+	 ('6c0079df-2647-445f-9718-4433e9761eac',NULL,NOW(),NULL,'migration_seed',NULL,'Village of Canal Flats','KOOR',NULL,false,true,'{}'),
+	 ('79c3e16e-fdb9-4bb0-a427-f044a5d4bb95',NULL,NOW(),NULL,'migration_seed',NULL,'District of Hudson''s Hope','NORR',NULL,false,true,'{}'),
+	 ('d3f74ba5-0dc3-43ea-a7e2-004882d65c33',NULL,NOW(),NULL,'migration_seed',NULL,'District of Barriere','INTR',NULL,false,true,'{}');
+
```
Comment (dhaselhan) [inline services/apps/alcs/src/providers/typeorm/migrations/1687546406729-seed_add_adl_loc_gov.ts:19]: random gap

File: services/apps/alcs/src/providers/typeorm/migrations/1687557623816-local_gov_name_edits.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`
+          UPDATE "alcs"."application_local_government" SET "name" = 'Village of Masset' WHERE "name" = 'Village of Massett';
+          DELETE FROM alcs.application_local_government WHERE name = 'Northern Rockies Regional Municipality';
+
+        `);
+        
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
```
Comment (dhaselhan) [inline services/apps/alcs/src/providers/typeorm/migrations/1687557623816-local_gov_name_edits.ts:14]: empty down

File: services/apps/alcs/src/providers/typeorm/migrations/1687810457499-seed_inactive_local_gov.ts [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+	 ('9355bf78-280a-4695-adad-fafa06969c99',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust-Capital (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('8017df89-8e71-41c3-bee9-1a48f042e157',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust-Powell River (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('26f395a1-44ab-4812-9e0d-f2dbc815859d',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust-Sunshine Coast (Historical)','SOUR',NULL,false,false,'{}'),
+	 ('b9056ee1-5ba3-4d7c-9029-2cec57ba75d5',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust - Cowichan Valley (Historical)','ISLR',NULL,false,false,'{}'),
+	 ('55f665bc-c91b-4bbb-85c2-39691088b297',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Bowen Island (Historical)','SOUR',NULL,false,false,'{}'),
+	 ('7577c7e9-d65b-4051-a0b1-bc95c462bdc3',NULL,NOW(),NULL,'migration_seed',NULL,'Islands Trust Denman Island (Historical)','ISLR',NULL,false,false,'{}');
+     `);
+    }
+    
+    public async down(queryRunner: QueryRunner): Promise<void> {
```
Comment (dhaselhan) [inline services/apps/alcs/src/providers/typeorm/migrations/1687810457499-seed_inactive_local_gov.ts:47]: empty down

**Pull request merged**